ExternalTest/rr_sim_tests/rr_sim_tests/utils/wait_for_spawned_entity.py

```diff
@@ -9,10 +10,10 @@
 
 SERVICE_NAME_GET_ENTITY_STATE = 'GetEntityState'
 
-def wait_for_spawned_entity(in_entity_name, in_timeout=10.0):
+def wait_for_spawned_entity(in_entity_name, service_namespace='', in_timeout=10.0):
```

Comment:
Ah we need to update
* [test_robot_spawn.py](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/yu_client_server_update/ExternalTest/rr_sim_tests/rr_sim_tests/test_robot_spawn.py#L98)
* [test_robot_remove.py](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/yu_client_server_update/ExternalTest/rr_sim_tests/rr_sim_tests/test_robot_remove.py#L74)
* [test_robot_spawn.py](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/yu_client_server_update/ExternalTest/rr_sim_tests/rr_sim_tests/test_robot_spawn.py#L98)
* [utils.py](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/yu_client_server_update/ExternalTest/rr_sim_tests/rr_sim_tests/utils/utils.py#L105)
* RapyutaSimInternal's [test_world_spawn.py](https://github.com/rapyuta-robotics/RapyutaSimulationInternal/blob/devel/ExternalTest/rr_sim_internal_tests/rr_sim_internal_tests/test_world_spawn.py#L115)

Reply:
I change the arg order to  avoid modification